Rai/Network/PriceController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  PriceController.swift
+//  Rai
+//
+//  Created by Ryan Fox on 12/22/17.
+//  Copyright © 2017 Zack Shapiro. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
```

Comment:
First, thanks for the PR!

I don't want to use SwiftyJSON, and should remove it from the Podfile. Swift 4's `Decodable` protocol is a much nicer way to encode/decode.

CMC's API has a rate limit and that's why I was avoiding it. Maybe write a fallback from Merc's API call to hit CMC if both initial APIs are offline?

Reply:
Yeah, I wasn't quite sure if you were moving towards SwiftJSON or away from it. Agreed that adopting as much as Swift gives us as possible is best!

As for CMC, their rate limits seem pretty generous -- one every six seconds. Would you expect to exceed that?